stdlib/2and3/argparse.pyi

```diff
@@ -140,13 +221,17 @@ class Action:
                  values: Union[_Text, Sequence[Any], None],
                  option_string: Optional[_Text] = ...) -> None: ...
 
-class Namespace:
+class Namespace(_AttributeHolder):
     def __init__(self, **kwargs: Any) -> None: ...
     def __getattr__(self, name: _Text) -> Any: ...
     def __setattr__(self, name: _Text, value: Any) -> None: ...
     def __contains__(self, key: str) -> bool: ...
 
 class FileType:
+    _mode: _Text
+    _bufsize: int
+    _encoding: Optional[_Text]
```

Comment:
Do these exist in Python 2? The construction signatures below suggests they don't.